runtime/nvqir/cutensornet/tensornet_state.cpp

```diff
@@ -440,6 +474,9 @@ TensorNetState::factorizeMPS(int64_t maxExtent, double absCutoff,
   HANDLE_CUTN_ERROR(cutensornetStateCompute(
       m_cutnHandle, m_quantumState, workDesc, extentsPtr.data(),
       /*strides=*/nullptr, allData.data(), 0));
+
+  if (hostWork)
+    free(hostWork);
```

Comment:
nit: Add braces? (I think the style in `runtime/` is to use braces as much as possible. :)

Reply:
oh, now I know :-) Fixed https://github.com/NVIDIA/cuda-quantum/pull/1732/commits/973ed7aef64bcb4b709dc85adeb1b27b2251ce4a